sdk/src/main/scala/com/horizen/storage/SidechainStateStorage.scala

```diff
@@ -53,38 +53,51 @@ class SidechainStateStorage (storage : Storage, sidechainBoxesCompanion: Sidecha
     }
   }
 
+  def getWithdrawalEpochInfo() : Option[WithdrawalEpochInfo] = {
```

Comment:
`SidechainStateStorage.get` method name should be reviewed. Previously we had the only method for retrieving boxes, now we also retrieve epoch info.
What do you think about renaming `SidechainStateStorage.get` to `SidechainStateStorage.getBox`